tests/unit/amo/server/test_base.js

```diff
@@ -165,10 +165,13 @@ describe(__filename, () => {
     });
 
     it('preconnects to addons-server CDN', async () => {
-      const config = getFakeConfig({
-        staticHost: undefined,
-        amoCDN: 'https://cdn.example.com',
-      });
+      const config = getFakeConfig(
+        {
+          staticHost: undefined,
```

Comment:
Isn't that the default already?

Reply:
ac284660baeab8985fc79a847a4e93739f26a5c3